docs/testing-notes.md

```diff
@@ -1,12 +1,66 @@
-# Build and deploy notes
+# Testing Notes
+
+## Build and deploy notes
+
+Use [Google Cloud Build](https://cloud.google.com/run/docs/building/containers#building_using) and [Container Registry](https://cloud.google.com/container-registry/docs/quickstart) to build and host the plugin docker image.
 
-* Use [Google Cloud Build](https://cloud.google.com/run/docs/building/containers#building_using) and [Container Registry](https://cloud.google.com/container-registry/docs/quickstart) to build and host the plugin docker image.
 ```shell
 $ export PROJECT_ID=<your gcp project>
 $ gcloud config set project $PROJECT_ID
 $ ./scripts/build.sh
+...
```

Comment:
what are these ellipses for?

Reply:
those represent the 'output' of the command, my markdown linter doesnt like `$` if you dont also put 'output'. I'll just remove both